### DISCUSSION TITLE
Welcome to staking-replicator Discussions!

### DISCUSSION THREAD

**Discussion by bradennapier** [idexio/staking-replicator#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by palettebuilders** [comment]
Hello, I have purchased 10K IDEX tokens and received my key.   I have little experience in creating a VPS staking node.   Is there a video example posted ? Or a chat channel i can join.   I do believe in this IDEX exchange.   I found IDEX through being an Algo hodler.  Your Discord link is broke
